docs/source/examples/petstore/petstore.all.rego

```diff
@@ -93,6 +100,11 @@ allow {
 
 # rego functions defined by seal
 
+# Helper to get the token payload.
+seal_subject = payload {
+	[header, payload, signature] := io.jwt.decode(input.subject)
```

Comment:
The jwt will come in as input.jwt, not input.subject; ie: io.jwt.decode(input.jwt)

Reply:
i fixed the description for #76 to use `input.jwt` - please make it so @abynenkov-ib

Reply:
please don't forget to fix the *.test.rego in the examples